lunary/anthrophic.py

```diff
@@ -0,0 +1,792 @@
+    metadata_parser=__metadata_parser,
+    contextify_stream: t.Optional[bool] = False,
+    *args,
+    **kwargs,
+):
+    output = None
+
+    parent_run_id = kwargs.pop("parent", None)
+    run = run_manager.start_run(run_id, parent_run_id)
+
+    with run_context(run.id):
+        try:
+            try:
+                params = params_parser(kwargs)
+                metadata = kwargs.get("metadata")
+                parsed_input = input_parser(kwargs)
+
+                if metadata:
+                    kwargs["metadata"] = metadata_parser(metadata)
+
+                track_event(
+                    type,
+                    "start",
+                    run_id=run.id,
+                    parent_run_id=parent_run_id,
+                    input=parsed_input["input"],
+                    name=name or parsed_input["name"],
+                    user_id=(kwargs.pop("user_id", None) or user_ctx.get() or user_id),
+                    user_props=(
+                        kwargs.pop("user_props", None)
+                        or user_props
+                        or user_props_ctx.get()
+                    ),
+                    params=params,
+                    metadata=metadata,
+                    tags=(kwargs.pop("tags", None) or tags or tags_ctx.get()),
+                    template_id=(
+                        kwargs.get("extra_headers", {}).get("Template-Id", None)
+                    ),
+                    is_openai=False,
+                )
+            except Exception as e:
+                return logging.exception(e)
+
+            if contextify_stream or kwargs.get("stream") == True:
+                return await stream_handler(
+                    method, run.id, name or parsed_input["name"], type, *args, **kwargs
+                )
+
+            try:
+                output = await method(*args, **kwargs)
+            except Exception as e:
+                track_event(
+                    type,
+                    "error",
+                    run.id,
+                    error={"message": str(e), "stack": traceback.format_exc()},
+                )
+                raise e from None
+
+            try:
+                parsed_output = output_parser(output, kwargs.get("stream", False))
+
+                track_event(
+                    type,
+                    "end",
+                    run.id,
+                    # In case need to compute tokens usage server side
+                    name=name or parsed_input["name"],
+                    output=parsed_output["output"],
+                    token_usage=parsed_output["tokensUsage"],
+                )
+                return output
+            except Exception as e:
+                logger.exception(e)(e)
+            finally:
+                return output
+        finally:
+            run_manager.end_run(run.id)
+
+
+if t.TYPE_CHECKING:
+    ClientType = t.TypeVar("ClientType")
+
+
+def monitor(client: "ClientType") -> "ClientType":
```

Comment:
You should extend the `monitor` function from __init__.py, and not redeclare a new one.
We want to do: `from lunary import monitor`, and that it works with both openai and Anthropic